transformations/negate_strengthen/transformation.py

```diff
@@ -0,0 +1,457 @@
+import random
+from interfaces.SentenceOperation import SentenceAndTargetOperation
+from tasks.TaskTypes import TaskType
+
+import re
+import spacy
+from spacy.tokenizer import Tokenizer
+import pandas as pd
+import nltk
+from nltk.corpus import wordnet
+from nltk.stem.wordnet import WordNetLemmatizer
+from nltk.metrics import edit_distance
+from nltk import pos_tag
+from pattern.en import conjugate as conjugate_en
+from typing import Union
+
+_RE_COMBINE_WHITESPACE = re.compile(r"\s+")
+MODAL_TYPE_LEMMA = ['be', 'to', 'have', 'get', 'do']
+MODAL_STRENGTHEN_DICT = {
+    'could': 'would',
+    'should': 'would',
+    'would': 'would',
+    'can': 'will',
+    'may': 'will',
+    'might': 'will',
+    'will': 'will'
+}
+# initialise
+nltk.download('wordnet')
```

Comment:
I think it's better to move these downloads into the constructor so that this is not called when the file is imported.

Reply:
Ok, done!